maven-scm-providers/maven-scm-provider-hg/src/main/java/org/apache/maven/scm/provider/hg/command/changelog/HgChangeLogConsumer.java

```diff
@@ -148,7 +148,6 @@ else if ( line.startsWith( MESSAGE_TOKEN ) )
         {
             StringBuilder comment = new StringBuilder( currentChange.getComment() );
             comment.append( line );
-            comment.append( '\n' );
```

Comment:
WTF? Who had this idea to modify the original comment?

Reply:
Removing this seemed the cleanest solution to me.